src/app/index.js

```diff
@@ -69,16 +69,22 @@ angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
       return $delegate;
     });
   })
-  .run(function($http, jhConfig) {
+  .run(function($rootScope,$http, jhConfig) {
+    //global function to replace the placeholders
+    $rootScope.replacePlaceholder = function(value){
```

Comment:
Why are you attaching this function to the `$rootScope`? What about just declaring and using the function? So you won't need to inject the `$rootScope` here.